client/landing/subscriptions/components/site-subscriptions-list/site-subscriptions-list.tsx

```diff
@@ -94,6 +110,25 @@ const SiteSubscriptionsList: React.FC< SiteSubscriptionsListProps > = ( {
 				<span className="email-frequency-cell" role="columnheader">
 					{ translate( 'Email frequency' ) }
 				</span>
+				{ isRecommendedBlogsEnabled && isLoggedIn && ! isCompactLayout && (
+					<span className="recommend-cell" role="columnheader">
+						{ translate( 'Recommend' ) }
+						<Tooltip
```

Comment:
`Tooltip` has some built-in a11y support, but tooltips aren't great for a11y or mobile. Open to suggestions for improvement.

Reply:
> A11y wise its in the keyboard navigation and pops up instantly on a11y-focus.

Thanks! You'll have to show me how you test that. 😅 

> Maybe an info link if we had a good recommended blogs page that described things, but perhaps the tooltip works best for now?

Yeah an info link sounds like a good next step. We can iterate from here. 🤞